gluten-ut/spark33/src/test/scala/org/apache/gluten/utils/clickhouse/ClickHouseTestSettings.scala

```diff
@@ -626,6 +626,10 @@ class ClickHouseTestSettings extends BackendTestSettings {
     .exclude("SPARK-36924: Cast YearMonthIntervalType to IntegralType")
     .exclude("SPARK-36924: Cast IntegralType to YearMonthIntervalType")
     .exclude("Cast should output null for invalid strings when ANSI is not enabled.")
+    .exclude("cast array from IntegerType to StringType")
+    .exclude("cast array from DoubleType to StringType")
+    .exclude("cast array from BooleanType to StringType")
+    .exclude("cast array from DateType to StringType")
```

Comment:
Why we need exclude these tests for CH backend? In theory, the execution should fall back for CH backend.